nat-lab/tests/test_pmtu.py

```diff
@@ -0,0 +1,199 @@
+import config
+import pytest
+from contextlib import AsyncExitStack
+from helpers import SetupParameters, setup_environment, setup_connections
+from telio import AdapterType
+from telio_features import TelioFeatures
+from utils import testing, stun
+from utils.connection_util import (
+    ConnectionTag,
+    generate_connection_tracker_config,
+    ConnectionLimits,
+)
+from utils.ping import Ping
+from utils.router import IPStack
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "setup_params",
+    [
+        pytest.param(
+            SetupParameters(
+                ip_stack=IPStack.IPv4,
```

Comment:
was it intended to be named `black_hole`? :)

Reply:
Um, yeah. The term refers to router not responding with ICMP errors

Reply:
I get it, but ther'es a typo, it's `back` hole instead of `black` :)

Reply:
Oh, haven't noticed that. Let me fix this

Reply:
Fixed